buf.gen.yaml

```diff
@@ -14,8 +14,7 @@ plugins:
     opt:
       - paths=source_relative
       - generate_unbound_methods=true
-  - remote: buf.build/jirkad/plugins/protoc-gen-validate
+  - plugin: buf.build/bufbuild/validate-go
```

Comment:
Newer versions of the `buf` CLI use the `plugin` key for both local and remote plugin references.

Also, we split the validate plugin into {go,cpp,java} so no need to add the `lang=go` plugin option